### ISSUE TITLE
Generating SRG maps on import spawns a separate gradle build for each subproject

### ISSUE THREAD

**Issue by thecodewarrior** [DemonWav/MinecraftDevIntelliJ#841]
* Minecraft Development for IntelliJ plugin version - 2021.1-1.5.4
* IntelliJ version  
    ```
    IntelliJ IDEA 2021.1 Beta (Ultimate Edition)
    Build #IU-211.6693.14, built on March 24, 2021
    ```
* Operating System - macOS Mojave, 10.14.6
* Target platform - ForgeGradle 4 (however the same issue occurred with FG3)

My project, [LibrarianLib](https://github.com/TeamWizardry/LibrarianLib) has a lot of gradle subprojects, 13 as of when I'm writing this. Whenever I reload the gradle build the plugin goes runs `createMcpToSrg` for each project individually. Given the size of the project, spooling up and configuring a gradle process takes some time, on top of the fact that it opens up 13 separate run tabs. It would be faster and less intrusive if the plugin could collect all the `createMcpToSrg` tasks in the project and run them all at once. (e.g. `gradlew :createMcpToSrg :core:createMcpToSrg :courier:createMcpToSrg`) This would have the benefit of only having to start up one gradle process (and likely being able to reuse the daemon) and only opening one tab in the run panel.

**Comment by DenWav** [comment]
This is a good idea, and Gradle would likely be able to re-use task outputs so it would reduce the workload. That being said, creating the mapping isn't the difficult part, as you said spinning up that many daemons is the problem.

I'll look into this and see how possible it is - the fix will be slightly more complicated due to how the Gradle callbacks work, but I'll see if I can figure out a nice way to condense all of the modules.

**Comment by thecodewarrior** [comment]
Awesome! The task outputs *are* actually reused, at least on a per project basis, so it essentially just spools up a gradle process, sees there's nothing to do, and exits.

**Comment by DenWav** [comment]
Force pushed the commit to edit the commit message, the one actually pushed which fixes this is 11ef1dbba86407681c6a62fb4ac439459318427f.

**Comment by thecodewarrior** [comment]
Awesome! Thanks!
